protostar/commands/test/test_cases.py

```diff
@@ -29,6 +29,7 @@ class PassedTestCase(TestCaseResult):
     test_case_name: str
     execution_resources: Optional[ExecutionResourcesSummary]
     captured_stdout: Dict[OutputName, str] = field(default_factory=dict)
+    fuzz_runs_count: int = 1
```

Comment:
I wonder if it wouldn't be more correct to make this

```suggestion
    fuzz_runs_count: Optional[int] = None
```

and keep it `None` for non-fuzz tests. The reason is that with current model, we lost an ability to track 1 fuzz test execution -- which although is not desirable, is totally possible if the test fails in second run.